pyproject.toml

```diff
@@ -25,18 +25,16 @@ pydantic = "^2.3.0"
 uvicorn = "^0.23.2"
 
 # Data Science
-jupyter = "^1.0.0"
 pandas = "^1.5.0"
 numpy = "^1.23.3"
+pyarrow = "^13.0.0"
```

Comment:
What is `pyarrow`? Your commit message gives no details nor does the code. I don't know what this dependency is or why we need it. Is it within the scope of `Continuous Deployment Pipeline for Raspberry Pi (Prediction Service)` or something else?

Reply:
`pyarrow` is needed to work with parquet files - `jupyter` as a dependency had pulled this in, but we can strip `jupyter` out (because we're not using it in dev or production) and just pull in `pyarrow` (a much smaller dependency).